.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: node_js
+node_js:
+  - "12"
+before_install:
+  - "sudo apt-get install -y libgnome-keyring-dev libsecret-1-dev"
+  - "sudo apt-get install -y xvfb"
+
+script: xvfb-run yarn test
+
+  # before_script:
+  # - "export DISPLAY=:99.0"
+  # - "sh -e /etc/init.d/xvfb start"
```

Comment:
It looks like tests are running, and this `before_script` block is not necessary.